cloud/google/configs/ct-mirror-certly1.sh

```diff
@@ -0,0 +1,9 @@
+export INSTANCE_TYPE="mirror"
+export PROJECT="ct-mirror-certly1"
```

Comment:
when did this log start being called 'certly1' ?  Haven't seen the number suffix before.